docs/modules/ROOT/pages/management-center-external-access.adoc

```diff
@@ -26,28 +26,30 @@ External connectivity defines how to expose Management Center:
 
 |===
 
-=== Example Management Center Ingress Configuration
+=== Example Management Center LoadBalancer Configuration
 
 The example configuration below does the following:
 
-- Expose Management Center using Ingress defined on the https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress].
+- Expose Management Center using a dedicated load balancer for your Management Center
 
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/management-center-ingress.yaml[]
+include::ROOT:example$/management-center-loadbalancer.yaml[]
 ----
 
-=== Example Management Center LoadBalancer Configuration
+=== Example Management Center Ingress Configuration
 
 The example configuration below does the following:
 
-- Expose Management Center using a dedicated load balancer for your Management Center
+- Expose Management Center using Ingress defined on the https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress].
 
 [source,yaml,subs="attributes+"]
 ----
-include::ROOT:example$/management-center-loadbalancer.yaml[]
+include::ROOT:example$/management-center-ingress.yaml[]
 ----
 
+WARNING: The paths, consisting of multiple segments (e.g., "/hazelcast/mancenter"), are supported as of Management Center version 5.4.0.
```

Comment:
```suggestion
WARNING: Paths that consist of multiple segments, such as _/hazelcast/mancenter_, are supported from Management Center version 5.4.0.
```

Reply:
Why is this a warning? Is anything broken by this change? What are the implications of this change?

Reply:
The complex paths like `/hazelcast/mancenter` will be supported after 5.4.0.

What is the best label for this message?

Reply:
NOTE: should be sufficient